subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt

```diff
@@ -85,6 +85,8 @@ class InstrumentationTestsPlugin : Plugin<Project> {
                 1
             ),
             prefixFilter = project.getOptionalStringProperty("dynamicPrefixFilter", ""),
+            skipSucceedTestsFromPreviousRun = project.getBooleanProperty("instrumentation.dynamic.skipSucceedTestsFromPreviousRun", true),
+            keepFailedTestsFromReport = project.getOptionalStringProperty("instrumentation.dynamic.keepFailedTestsFromReport"),
```

Comment:
Maybe we should name it "run" instead of "keep"?

Reply:
I think it no guaruntees that it will be run. Keep is better for filter

Reply:
I see. You're right